data/mods/Aftershock/mutations/mutations.json

```diff
@@ -136,7 +136,7 @@
     "visibility": 3,
     "ugliness": 6,
     "description": "You can see the connections from A to B to C to D.  You look at fellow survivors like you are imagining taking them apart.  Intelligence + 5",
```

Comment:
I'm just waking up, but - is there a GUILT-reducing trait that should also be a prerequisite for this one?

Reply:
There is not but that is a great idea. I hadn't thought of it as cannibalism as much as mechanically, vitruvian man style.

Reply:
> There is not but that is a great idea. I hadn't thought of it as cannibalism as much as mechanically, vitruvian man style.

Thanks! Perhaps either of Psychopath or Cannibal? The first might be added to Mi-Go category mutations.